plugin/trino-druid/src/test/java/io/trino/plugin/druid/TestDruidCaseInsensitiveMatch.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.druid;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.jdbc.mapping.TableMappingRule;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.MaterializedResult;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import static io.trino.plugin.druid.DruidQueryRunner.copyAndIngestTpchDataFromSourceToTarget;
+import static io.trino.plugin.druid.DruidQueryRunner.createDruidQueryRunnerTpch;
+import static io.trino.plugin.druid.DruidTpchTables.SELECT_FROM_ORDERS;
+import static io.trino.plugin.druid.DruidTpchTables.SELECT_FROM_REGION;
+import static io.trino.plugin.jdbc.mapping.RuleBasedIdentifierMappingUtils.createRuleBasedIdentifierMappingFile;
+import static io.trino.plugin.jdbc.mapping.RuleBasedIdentifierMappingUtils.updateRuleBasedIdentifierMappingFile;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+import static io.trino.tpch.TpchTable.ORDERS;
+import static io.trino.tpch.TpchTable.REGION;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Test(singleThreaded = true)
+public class TestDruidCaseInsensitiveMatch
+        extends AbstractTestQueryFramework
```

Comment:
Unfortunate that we aren't using `BaseCaseInsensitveNameMatchingTest` instead - it has test-cases for all the issues we found over time in this feature.

I think we did add some way to ingest data into Druid for tests - see https://github.com/trinodb/trino/pull/13335/files

Reply:
The problem of extending BaseCaseInsensitveNameMatchingTest is that Druid does not support any other schema name than `druid` (kinda related to https://github.com/trinodb/trino/issues/14700)  . So it felt over complicated and unnecessary to override all the schemaName case sensitive tests where it only supports the `druid` schema.

Reply:
extended BaseCaseInsensitveNameMatchingTest